### PR TITLE
feat: [SIW-1362] Update backend status with new remote config

### PR DESCRIPTION
**Pull request by hevelius**
## Short description
This PR updates content definitions to version `1.0.35` and add new backendStatus configuration for itw.

## List of changes proposed in this pull request
- Update package.json
- Update backend.ts

## How to test
Start the app and check if backstatus contains itw config section. The io-app should use version `1.0.35` of content definitions

### CODE REVIEW

**Review by shadowsheep1 — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   "api_cgn": "https://raw.githubusercontent.com/pagopa/io-backend/v13.39.1-RELEASE/api_cgn.yaml",
   "api_cgn_merchants": "https://raw.githubusercontent.com/pagopa/io-backend/v13.39.1-RELEASE/api_cgn_operator_search.yaml",
   "api_cgn_geo": "https://raw.githubusercontent.com/pagopa/io-backend/here_geoapi_integration/api_geo.yaml",
-  "content_specs": "https://raw.githubusercontent.com/pagopa/io-services-metadata/1.0.32/definitions.yml",
+  "content_specs": "https://raw.githubusercontent.com/pagopa/io-services-metadata/1.0.35/definitions.yml",
```
Comment (Vangaorth) [inline package.json:12]: Shall we keep all io-services-metadata versions to 1.0.35?

**Review by Vangaorth — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
...
   "api_pagopa_walletv2": "https://raw.githubusercontent.com/pagopa/io-services-metadata/1.0.30/bonus/specs/bpd/pm/walletv2.json",
   "api_pagopa": "https://raw.githubusercontent.com/pagopa/io-app/master/assets/paymentManager/spec.json",
   "pagopa_cobadge_configuration": "https://raw.githubusercontent.com/pagopa/io-services-metadata/1.0.30/pagopa/cobadge/abi_definitions.yml",
```
Comment (shadowsheep1) [inline package.json:15]: What about updating all the `io-services-metadata` references?

**Review by Vangaorth — APPROVED** [review]
LGTM

**Pull request merged**